src/plugins/intel_cpu/src/nodes/mkldnn_deconv_node.cpp

```diff
@@ -604,89 +736,128 @@ void MKLDNNDeconvolutionNode::prepareParams() {
+                                                                   key.inp0->getDnnlDesc(),
+                                                                   key.inp1->getDnnlDesc(),
+                                                                   key.out->getDnnlDesc(),
+                                                                   engine);
+                } else {
+                    auto prim_desc = convolution_backward_data::primitive_desc(itpd.get());
+                    execPtr = std::make_shared<DeconvExecutorDefault>(prim_desc,
+                                                                      key.inp0->getDnnlDesc(),
+                                                                      key.inp1->getDnnlDesc(),
+                                                                      key.out->getDnnlDesc(),
+                                                                      engine);
+                }
+            }
+        }
 
-    auto fwd_conv_pd = std::make_shared<convolution_forward::primitive_desc>(*conv_desc, getEngine(), true);
+        return execPtr;
+    };
 
-    return {deconv_desc, fwd_conv_pd};
-}
+    execPtr = nullptr;
+    auto cache = getRuntimeCache();
+    auto result = cache->getOrCreate(key, builder);
 
-MKLDNNDeconvolutionNode::Int8DeconvDesc MKLDNNDeconvolutionNode::createDescriptorInternalInt8(const mkldnn::memory::desc& in_candidate,
-                                                                                                   const mkldnn::memory::desc& wgh_candidate,
-                                                                                                   const mkldnn::memory::desc& out_candidate) const {
-    auto convertDims = [] (const std::vector<ptrdiff_t>& orig_dims) {
-        return memory::dims(orig_dims.begin(), orig_dims.end());
-    };
+    execPtr = result.first;
 
-    MKLDNNDeconvolutionNode::Int8DeconvDesc deconv_desc;
-    deconv_desc = std::make_shared<mkldnn::deconvolution_forward::desc>(prop_kind::forward_inference, mkldnn::algorithm::deconvolution_direct,
-                                                                        in_candidate, wgh_candidate, out_candidate,
-                                                                        convertDims(stride), convertDims(dilation),
-                                                                        convertDims(paddingL), convertDims(paddingR));
-    return deconv_desc;
+    if (execPtr) {
+        if (key.isInt8) {
+            primArgs = {{DNNL_ARG_SRC, srcMemPtr->GetPrimitive()},
+                        {DNNL_ARG_WEIGHTS, internalBlobMemory.front()->GetPrimitive()},
+                        {DNNL_ARG_DST, dstMemPtr->GetPrimitive()}};
+        } else {
+            primArgs = {{DNNL_ARG_DIFF_DST, srcMemPtr->GetPrimitive()},
+                        {DNNL_ARG_WEIGHTS, wghMemPtr->GetPrimitive()},
+                        {DNNL_ARG_DIFF_SRC, dstMemPtr->GetPrimitive()}};
```

Comment:
Minor. It is better to rewrite using `[]` operator.

Reply:
fixed